### PR TITLE
feat: all others cities added; pagination added in location and searc…

### PR DESCRIPTION
**Pull request by Xayrulloh**
all cities count is 89
if inline keyboard length more then 12 then implemented to pagination
pagination changes in the scenes which has more then 12 keyboards
inlineKFunction takes page param

### CODE REVIEW

**Review by eugene0928 — APPROVED** [review]

File: keyboard/inline.ts [hunk 1]
```diff
@@ -5,15 +5,32 @@ type InternalButton = {
...
   let keyboard = new InlineKeyboard()
 
-  buttons.forEach((el, index) => {
-    if (index % num == 0) {
-      keyboard.row()
-    }
-    keyboard.text(el.view, el.text)
-  })
+  if (buttons.length >= 15) {
+    const displayButtons = buttons.splice(page * 12 - 12, 12)
```
Comment (toshmukhamedov) [inline keyboard/inline.ts:12]:
```suggestion
    const displayButtons = buttons.splice((page - 1) * 12, 12)
```

File: keyboard/inline.ts [hunk 2]
```diff
@@ -5,15 +5,32 @@ type InternalButton = {
   text: string
 }
 
-export default function inlineKFunction(num: number, ...buttons: InternalButton[]) {
+export default function inlineKFunction(num: number, page: number, ...buttons: InternalButton[]) {
```
Comment (toshmukhamedov) [inline keyboard/inline.ts:8]:
```suggestion
export default function inlineKFunction(num: number, buttons: InternalButton[], page = 1) {
```
qogan joylada ham shunchaki array berish kerak, pageni olib tashlab

File: translate/uz.json [hunk 3]
```diff
@@ -1,18 +1,94 @@
 {
   "region": {
     "Toshkent": 27,
+    "Samarqand": 18,
     "Andijon": 1,
```
Comment (toshmukhamedov) [inline translate/uz.json:5]: `ASC` bo’yicha tartiblasak ajoyib bo’lar edi

**Review by toshmukhamedov — CHANGES_REQUESTED** [review]

File: keyboard/inline.ts (hunk 2 above)
Comment (Xayrulloh) [reply keyboard/inline.ts:8]: agree, but I can't suggest it like this now, I've to change it from other places that called this function

File: translate/uz.json (hunk 3 above)
Comment (Xayrulloh) [reply translate/uz.json:5]: first 12 is by famous, then alphabetic

**Review by toshmukhamedov — APPROVED** [review]

**Pull request merged**